### ISSUE TITLE
Prometheus + Grafana Integration not working

### ISSUE THREAD

**Issue by QAInsights** [tsenart/vegeta#662]
#### Version and Runtime

```
Version: 12.11.1
Commit: 
Runtime: go1.21.3 darwin/arm64
Date: 2023-10-02T09:03:39Z
```

#### Expected Behaviour

`echo "GET http://localhost/" | vegeta attack -duration=15m -prometheus-addr 0.0.0.0:9090` command works fine. I can see the prometheus stats in the browser. But the Grafana integration is failing.

#### Actual Behaviour

While trying to add Prometheus data source in Grafana showing below error.
![image](https://github.com/tsenart/vegeta/assets/2826376/db17de5e-3717-4a35-bd67-224ebc7f3b14)

#### Steps to Reproduce

1. Install vegeta
2. Install grafana
3. Run vegeta w/ prometheus flag
4. Add prometheus datasource in Grafana

#### Additional Context

`grafana version 10.2.0`

**Comment by tsenart** [comment]
That seems like an issue between Grafana and Prometheus. Or am I misunderstanding?

**Comment by QAInsights** [comment]
Yes, that's correct. I reinstalled everything starting from scratch. Started the test in vegeta. But the dashboard is blank. 

![image](https://github.com/tsenart/vegeta/assets/2826376/a8abd351-be99-4198-b16e-a0680d67bfa1)

But the Grafana dashboard looks empty.
<img width="1443" alt="image" src="https://github.com/tsenart/vegeta/assets/2826376/6739d375-4fcf-43ef-8169-f7227438bcf8">

Prometheus data source
![image](https://github.com/tsenart/vegeta/assets/2826376/5901f372-4f7d-428a-8024-2911f7d1a1ec)

**Comment by ocodista** [comment]
It seems that adding the -prometheus-addr is not pushing any data to prometheus, I wonder why would that be.
